braindecode/preprocessing/preprocess.py

```diff
@@ -265,6 +264,7 @@ def zscore(data):
         If this function is supposed to preprocess continuous data, it should be
         given to raw.apply_function().
     """
+    warn('zscore is deprecated. Use sklearn.preprocessing.scale instead.')
```

Comment:
you should warn  with DeprecationWarning and UserWarning

Reply:
Ah nice, let's use `mne.utils.deprecated` then! However it looks like it will only warn with `DeprecationWarning`, not `UserWarning`?